x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/schema.tsx

```diff
@@ -246,6 +246,12 @@ export const schema: FormSchema<DefineStepRule> = {
   },
   relatedIntegrations: {
     type: FIELD_TYPES.JSON,
+    label: i18n.translate(
+      'xpack.securitySolution.detectionEngine.createRule.stepAboutRule.fieldRelatedIntegrationsLabel',
+      {
+        defaultMessage: 'Related integrations',
```

Comment:
Fixes a missing label in create rule preview
<img width="688" alt="Scherm­afbeelding 2024-05-07 om 12 15 04" src="https://github.com/elastic/kibana/assets/15949146/aaedf303-8189-410a-a462-e1b685692fb4">